ci/citest/acceptance-test/multibox/10.0.100.16-vdc-executor-qemu/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+export ENV_ROOTDIR="$(cd "$(dirname $(readlink -f "$0"))/.." && pwd -P)"
+export NODE_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TMP_ROOT="${NODE_DIR}/tmp_root"
+
+. "${ENV_ROOTDIR}/config.source"
+. "${NODE_DIR}/vmspec.conf"
+. "${ENV_ROOTDIR}/ind-steps/common.source"
+
+IND_STEPS=(
+    "box"
+    "ssh"
+    "hosts"
+    "disable-firewalld"
+    "epel"
+    "qemu"
+    "kvm"
+    "ncat"
+    "bridge-utils"
+    "mesosphere-repo"
+    "mesos-agent"
+    "vdc-images"
```

Comment:
I think it is better to have only one image server in the CI environment instead of having it locally on multiple executors. That seems more likely in a production environment and tests our capability to get images from a remote location.

Edit: This is me assuming that we have a web server serving images for the executors. Please correct me if I'm wrong.